src/internal/components/ThemeEditor.tsx

```diff
@@ -95,6 +96,7 @@ export const ThemeEditor = (props: ThemeEditorProps) => {
         setThemeInitialHistory({
           history: localHistories,
           index: localHistoryIndex,
+          hash: uuidv4(),
```

Comment:
We need to update what the theme history contains. It needs to also include the hash. Hashes should only be generated when a new theme history is added. Since it was created and put into localStorage, we can't set new hashes when pulling it back out. Otherwise we'll never be able to find the correct history based on hash.